### PR TITLE
produce an unsampled, non-temporal build

### PR DESCRIPTION
**Pull request by jameshadfield**
@huddlej @rneher and others -- here is the branch I'm using to produce non-subsampled builds. As you can see there are a few modifications i'm making to the snakemake file. Are there ways we can streamline this? I could think of a couple but am unsure the best practices here:
* A `config["refine"]["infer_temporal_structure"]` (bool) could be defined, and the snakemake rule `refine` calls `augur refine` with the appropriate parameters. 
* A `config["refine"]["traits"]["run"]` (bool) could be set and the `export` rule updated to dynamically use the appropriate node-data JSONs
* The change to `scripts/modify-tree-according-to-exposure.py` should probably be in master

Are there cleverer ways to accomplish this? (I'm also happy keeping a parallel branch, but it gets tedious...)

### CODE REVIEW

**Comment by rneher** [comment]
we could implement this in the context of the `generic-subsampling` branch. as a flag to the `config['builds']`

**Comment by jameshadfield** [comment]
Will revisit this PR after the merge of #388

File: rules/builds.smk [hunk 1]
```diff
@@ -311,18 +311,21 @@ rule tree:
...
+        mem_mb=lambda wildcards, input: 40 * int(input.size / 1024 / 1024),
+    params:
+        mem_gb=lambda wildcards, resources: int(resources.mem_mb / 1024)-1,
     conda: config["conda_environment"]
     shell:
         """
         augur tree \
             --alignment {input.alignment} \
             --output {output.tree} \
+            --tree-builder-args="-mem {params.mem_gb}G" \
```
Comment (jameshadfield) [inline rules/builds.smk:328]: This doesn't work as expected (at least, not on AWS) where we get 
```
Job FAILED after 290.4 minutes (OutOfMemoryError: Container killed due to memory usage, exited 1)
```
Despite with iq-tree being run with `-mem 30G` and snakemake & AWS given 40Gb.

Comment (tsibley) [reply rules/builds.smk:328]: Seems worth trying a node size with a much larger memory allocation, making sure the rule is benchmarked, and then seeing what is actually used to guide node size choice.

Comment (jameshadfield) [reply rules/builds.smk:328]: Will do -- running now on a node with 64Gb (confusingly, `params.mem_gb` is 32, not sure why).

**Comment by rneher** [comment]
impossible by now.

**Pull request closed without merging**